rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -279,12 +282,43 @@ def _create_snapshot_with_invalid_headers(directory: str, timestamp: Timestamp)
         _write_location_data_csv_row_with_invalid_headers(writer, timestamp)
 
 
+def validate_timestamp(
+        received_serialized_timestamp: str,
+        expected_utc_timestamp: Timestamp,
+        display_date_in_localtime: bool,
+        is_for_import: bool,
+):
+    """Validate exported timestamp.
+
+    If is_for_import is True, then the timestamp should be a utc timestamp in seconds.
+    If display_date_in_localtime is True, then timestamp should be stringified in local timezone.
+    Otherwise the timestamp should be stringified in utc.
+    """
+    if is_for_import:  # Should always be a numeric utc timestamp
+        deserialized_timestamp = deserialize_timestamp(received_serialized_timestamp)
+        assert deserialized_timestamp == expected_utc_timestamp
+    else:
+        if display_date_in_localtime:  # Check that respects the timezone
```

Comment:
```suggestion
        if display_date_in_localtime:  # Check that it respects the timezone
```

Reply:
it is all good, you can skip and proceed. @LefterisJP 

@nebolax when you fix the above suggestion, push again, and let's see.